clusterDeployer.py

```diff
@@ -489,19 +495,9 @@ def _rename_workers(self, infra_env: str) -> None:
         ip_range = self._cc.full_ip_range
         logger.info(f"Connectivity established to all workers; checking that they have an IP in range: {ip_range}")
 
-        def any_address_in_range(h: host.Host, ip_range: tuple[str, str]) -> bool:
-            for ipaddr in common.ip_addrs(h):
-                for ainfo in ipaddr.addr_info:
-                    if ainfo.family != "inet":
-                        continue
-                    if not common.ip_range_contains(ip_range, ainfo.local):
-                        continue
-                    return True
-            return False
-
         any_worker_bad = False
         for w, h in zip(workers, hosts):
-            if not any_address_in_range(h, ip_range):
+            if not common.any_address_in_range(h, ip_range):
```

Comment:
rewrite ip_range as a class so that you can call `ip_range.any_address_inside(h)`

Reply:
I think that's a bit out of scope of this PR, I can add it if you insist, I'm just not sure how the API for that would look like

Reply:
Ok, we can skip that in this PR